Doc/faq/design.rst

```diff
@@ -151,66 +151,10 @@ to tell Python which namespace to use.
 Why can't I use an assignment in an expression?
 -----------------------------------------------
 
-Many people used to C or Perl complain that they want to use this C idiom:
+Starting in Python 3.8, you can! There is a new syntax, `:=`, that assigns a
```

Comment:
@emilyemorehouse 
Maybe with this sentence?
```
Starting in Python 3.8, you can use the walrus operator `:=` that assigns a variable in an expression::

    while chunk := fp.read(200):
        print(chunk)
```

Reply:
Thanks! I used your example and added something similar.